code/modules/client/preference/preferences.dm

```diff
@@ -1194,7 +1194,7 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 		else if(href_list["select_category"])
 			gear_tab = href_list["select_category"]
 		else if(href_list["clear_loadout"])
-			gear.Cut()
+			gear = null
```

Comment:
This is the only loadout modifying thing here, and it just clears it which shouldnt case a list runtime. I am so confused.

Reply:
gear = null clears the data type wheras .cut makes it a list with 0 entries, I assume. So if a later code does something like
gear += newgear_item then it will actually set gear to be OF type newgear_item, rather than a list with just a single entry, which is newgear_item. A similar issue broke mentor/admin tickets by adding tickets to the list before it was initialized

Reply:
`gear` is never defined as a list though, its a single item.
```
			var/datum/gear/gear = GLOB.gear_datums[href_list["gear"]]  <----------
			var/datum/gear_tweak/tweak = locate(href_list["tweak"])
			if(!tweak || !istype(gear) || !(tweak in gear.gear_tweaks))
				return
			var/metadata = tweak.get_metadata(user, get_tweak_metadata(gear, tweak))
			if(!metadata || !CanUseTopic(user))
				return
			set_tweak_metadata(gear, tweak, metadata)
		else if(href_list["select_category"])
			gear_tab = href_list["select_category"]
		else if(href_list["clear_loadout"])
			gear = null <----------
```

Reply:
Check line 203
`	//Gear stuff
	var/list/gear = list()`
Also the code just above the bit you commented does obviously use it as a list, ie for(var/gear_name in gear)